string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,8 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  var allLowerCase = word.toLowerCase();
+  if (word === allLowerCase) {
```

Comment:
You can also handle all of this is one line, because conditions will always evaluate to either true or false  
`return word === word.toLowerCase()`